app/api/tenants/tenantContext.ts

```diff
@@ -16,13 +16,11 @@ export type Tenant = {
 class Tenants {
   storage = new AsyncLocalStorage<string>();
 
-  defaultTenantName = 'default';
-
   tenants: { [k: string]: Tenant };
 
   constructor() {
     this.tenants = {
-      [this.defaultTenantName]: { name: this.defaultTenantName, ...config.defaultTenant },
+      [config.defaultTenant.name]: config.defaultTenant,
```

Comment:
Why do we force to always have a "default" tenant?  If I configure two named tenants, would this keep ALSO a "default" tenant that no one asked for?

Reply:
probably yes, can you create a separate issue to discuss this if you think it merits as it has nothing to do with the hotfix.
as a possible solution we can set the tenants array to blank if there is some tenant coming from the tenants change event.